### PR TITLE
Feature educational

### PR DESCRIPTION
**Pull request by Soy-yo**

### CODE REVIEW

**Review by Inmapg — APPROVED** [review]

File: bitcoinMiner.txt [hunk 1]
```diff
@@ -1,958 +1,1484 @@
...
+        Para añadir un cartelito educacional nuevo copia el anterior, cambia el id, pon el texto que quieras
+        y modifica eduBack y eduNext para que se muestre en el orden correcto. Y las líneas hide eduBack,
+        show eduNext, etc., tienen que estar siempre en el primer y último if de la forma en que están ahora.
+    */
+    
+    *eduBack
+        name:Back
+        tag:eduControl
+        class:edu-back
+        show clicks
```
Comment (ManuKai1) [inline bitcoinMiner.txt:540]: Why? Change it to -> no tooltip

File: bitcoinMiner.txt [hunk 2]
```diff
@@ -1,958 +1,1484 @@
...
+                show eduNext
+            end
+        end
+        hidden
+    
+    *eduNext
+        name:Next
+        tag:eduControl
+        class:edu-next
+        show clicks
```
Comment (ManuKai1) [inline bitcoinMiner.txt:655]: Why? Change it to -> no tooltip

File: bitcoinMiner.txt [hunk 3]
```diff
@@ -1,958 +1,1484 @@
...
+	*TEMPLATE
+		on click:anim glow
+ 
+	//https://2cryptocalc.com/most-profitable-gpu https://www.nicehash.com/profitability-calculator
+	//The real cost of GPU is between $400-$1800
+	//It produces 0.00024400BTC/day ==> $4.74
+  
+  *pickaxe|pickaxes
+  	name:Pickaxe|Pickaxes
+    desc:A pickaxe will autoclick once every second.
```
Comment (ManuKai1) [inline bitcoinMiner.txt:966]: I'll add the correct description (with effect) here

**Comment by lauracastilla** [comment]
Very well Manuel

**Pull request merged**